.golangci.yml

```diff
@@ -0,0 +1,118 @@
+run:
+  timeout: 10m
+  skip-files:
+    - "zz_generated.*.go"
+    - ".*_test.go"
+
+linters-settings:
+  gomodguard:
+    blocked:
+      modules:
+        - github.com/go-logr/logr:
+            recommendations:
+              - k8s.io/klog/v2
+  gci:
+    local-prefixes: github.com/liqotech/liqo
+  goconst:
+    min-len: 2
+    min-occurrences: 2
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - opinionated
+      - performance
+      - style
+    disabled-checks:
+      # Conflicts with govet check-shadowing
+      - sloppyReassign
+  goimports:
+    local-prefixes: github.com/liqotech/liqo
+  golint:
+    min-confidence: 0.8
+  govet:
+    check-shadowing: true
+  misspell:
+    locale: US
+  nolintlint:
+    allow-leading-space: true # don't require machine-readable nolint directives (i.e. with no leading space)
+    allow-unused: false # report any unused nolint directives
+    require-explanation: true # require an explanation for nolint directives
+    require-specific: true # require nolint directives to be specific about which linter is being skipped
+  dupl:
+    threshold: 300
+
+linters:
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - errcheck
+    - errorlint
+    - exhaustive
+  # - funlen
+  # - gochecknoglobals
+  # - gochecknoinits
+  # - gocognit
+    - goconst
+    - gocritic
+    - gocyclo
+    - godot
+  # - godox
+  # - goerr113
+    - gofmt
+    - goimports
+    - golint
+    - gomodguard
+  # - gomnd
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+  # - lll
```

Comment:
I would personally enable lll, to prevent excessively long lines (probably 120 is a good compromise)